cypress/integration/ColorPicker.spec.ts

```diff
@@ -129,48 +125,52 @@ describe('ColorPicker', () => {
       });
 
       context('when a swatch is clicked', () => {
-        const color = '8660d1';
+        const color = '#ff5347';
+
         beforeEach(() => {
-          cy.get(`.wdc-color-picker--color-${color}`).click();
-          cy.get('button').click();
+          getSwatch(color).click();
+          getOpenButton().click();
         });
 
         it('should have check icon', () => {
-          cy.get(`.wdc-color-picker--color-${color}`)
+          getSwatch(color)
             .find('.wd-icon')
             .should('exist');
         });
-      });
 
-      context('when color reset is clicked', () => {
-        beforeEach(() => {
-          cy.get(`.wdc-color-picker--color-8660d1`).click();
-          cy.get('button').click();
-          cy.get('[data-testid="color-picker-reset"]').click();
-        });
+        context('when color reset is clicked', () => {
```

Comment:
Nested this one a level deep, we shouldn't have to click on a swatch again when we can just put it under the `when swatch is clicked` context.

Did this below in the "popup" version of this test.